vpr/src/pack/output_clustering.cpp

```diff
@@ -486,8 +491,8 @@ static void clustering_xml_block(pugi::xml_node parent_node, t_type_ptr type, t_
 				node_index = pb->pb_graph_node->clock_pins[port_index][j].pin_count_in_cluster;
 				if (pb_type->parent_mode == nullptr) {
                     if (pb_route.count(node_index)) {
-                        pins.push_back(clustering_xml_net_text(pb_route[node_index].atom_net_id));
-                    } else {
+					pins.push_back(clustering_xml_net_text(pb_route[node_index].atom_net_id));
+				} else {
```

Comment:
This is a formatting error, should revert

Reply:
Fixed